pom.xml

```diff
@@ -42,6 +42,8 @@
 
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
@kwhetstone As I mentioned, I prefer to remove this property `<findbugs.failOnError>true</findbugs.failOnError>`